### PR TITLE
chore: Move send email into util function and handle users with no email client setup

### PR DESCRIPTION
**Pull request by annacarey**
The type of this PR is: **Enhancement**

This PR resolves [CX-561]

### Description

Moved email functionality into a util method and added a check to see if email client is setup on user's device. If it's not set up, user will get a notification saying that no email client is set up and to email the address proved as a method argument directly. 

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-561]: https://artsyproduct.atlassian.net/browse/CX-561

### CODE REVIEW

**Review by pvinis — APPROVED** [review]
🙌🙌🙌

**Comment by pvinis** [comment]
#squashongreen

**Comment by annacarey** [comment]
> #squashongreen

did this work?

**Comment by pvinis** [comment]
hm. no idea. I'll commit the suggestion and then I'll merge manually.

File: CHANGELOG.yml [hunk 1]
```diff
@@ -2,7 +2,9 @@ upcoming:
   version: 6.7.6
   date: TBD
   dev:
-    -
+    - Analytics tracking for My Collection - pepopowitz, annacarey
+    - Fix `isVisible` root view prop to respect tab switches - david, erik, lily
```
Comment (pvinis) [inline CHANGELOG.yml:6]:
```suggestion
```

**Review by pvinis — APPROVED** [review]

**Comment by pvinis** [comment]
should be alright now

File: CHANGELOG.yml [hunk 2]
```diff
@@ -2,7 +2,9 @@ upcoming:
...
+    - Surface alert message for users with no email client setup - annacarey, pavlos
```
[the remark this answers is not in the document]
Comment (annacarey) [reply CHANGELOG.yml:7]: I think i just fixed it!

**Pull request merged**